content/blog/urbit-and-bitcoin.md

```diff
@@ -0,0 +1,17 @@
++++
+title = "Urbit and Bitcoin"
+description = "A sound money deserves a sound computer."
+date = 2019-10-17
+[extra]
+author = "Christian Langalis"
+ship = "~pindet-timmut"
++++
+
+Urbit imagines a new model for an OS that – among other changes – embeds money as a primitive in the system’s architecture. Developers within the Urbit ecosystem will be able to securely use money in their applications as easily as they might use file systems or networking protocols. Creating a new sound currency, however is not our goal; Bitcoin has already done that.  
```

Comment:
> Urbit ...  embeds money as a primitive in the system’s architecture

This isn't true.  It'll be very natural to use money from Urbit, especially cryptocurrencies.  But it's not a primitive in any sense.

Reply:
Fair point, it's not *presently* the case; it's meant to be aspirational, I think, but I didn't write or edit the piece. @galenwp and @eriknewton should comment.

Reply:
I'm down with the "prophetic present" tense, but I don't know in what sense it'll be a primitive even in the future.

Reply:
Agreed that it seems unlikely for "money" to be a primitive at the system level. We'll probably have an affordance for "pay ~friend $x value, don't care in what currency", but I imagine that living entirely in userspace, not as _part of the system_.

Reply:
This is pedantic criticism. Userspace infrastructure is certainly a part of the system and can provide primitives.

Reply:
System != kernel

Reply:
Userspace can be part of the system, but "embeds money as a primitive in the system's architecture" says that money has some relevance to the architecture of the system, which it doesn't.

This is only pedantry if it doesn't change what the sentence communicates.  If I see an operating system that "embeds money as a primitive", I assume it's a scam because there's no reason to do that.  It signals that the whole thing is just a money-grab, like when Keybase "embedded" Stellar into their system.

Reply:
Maybe something like "Urbit imagines a new model for an OS that includes money as a native service."

Reply:
Noted. I agree that “primitive in system’s architecture” is untrue. “primitive in the system” does not have the same issue imo, but I can see arguments against that. “native service” does communicate the same thing, but not as strongly as I’d like.

Reply:
"tightly woven into the experience of"?

Reply:
See #243.